manifests/f/FastStone/Viewer/7.6/FastStone.Viewer.locale.en-US.yaml

```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: FastStone.Viewer
+PackageVersion: "7.6"
+PackageLocale: en-US
+Publisher: FastStone Corporation
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: FastStone Image Viewer
+PackageUrl: https://www.faststone.org/FSViewerDetail.htm
+License: Copyright (c) 2020 FastStone Soft
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate